README.md

```diff
@@ -369,7 +369,8 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ active community contributors.
+- read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```

Comment:
this can't really be a bullet point under "the lightning community is maintained by ..."